src/main/java/ReviewMyCodeApplication.java

```diff
@@ -0,0 +1,16 @@
+import jakarta.annotation.PostConstruct;
```

Comment:
The import should be 'javax.annotation.PostConstruct' instead of 'jakarta.annotation.PostConstruct'.